application/templates/pages/guidance/view-your-data.html

```
@@ -0,0 +1,22 @@
+{%- do pageData.update({
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)